susie/src/main/scala/MakeSuSiE.scala

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.dig.aggregator.methods.susie
+
+import org.broadinstitute.dig.aggregator.core._
+import org.broadinstitute.dig.aws.emr._
+import org.broadinstitute.dig.aws.Ec2.Strategy
+import org.broadinstitute.dig.aws.MemorySize
+
+class MakeSuSiE(implicit context: Context) extends Stage {
+  import MemorySize.Implicits._
+
+  val ancestrySpecific: Input.Source = Input.Source.Success("out/metaanalysis/bottom-line/ancestry-clumped/*/*/")
```

Comment:
For now I guess we'll just mess with EU data since you only have EU data right. Replace the second '*' with `ancestry=EU` and that should make it specific

Reply:
done!